### ISSUE TITLE
[Shaman-Enh] Windfury Totem

### ISSUE THREAD

**Issue by ewer0012** [herotc/hero-rotation#686]
When you have another Enh Shaman in the group, if they drop their Windfury totem before you, it only shows that you should drop Windfury totem, till your totem is the last one standing. Is there a way to get an option to show totem reminders on the side?

**Comment by ewer0012** [comment]
Or, I guess, have it pick up that you have a WF totem buff, regardless of what Enh Shaman it's from?

**Comment by lithium720** [comment]
should be fixed in c3aa01c

**Comment by ewer0012** [comment]
I made this change in my Enhancement.lua but it doesn't seem to fix the issue. Is there another change that needs to be made for it to work?

With 2+ Enh Shaman in the group, if they place their totem(s) first, It just shows that I need to cast Windfury Totem and no other spells.

Tested this out with another Shaman on the training dummies just a few minutes before posting. The sequence was:

1: Other Shaman dropped WF Totem
2: I dropped WF Totem for my Doom Winds buff
3: Hero Rotation only shows that I should drop WF Totem, even though I have my WF totem down and I have my Doom Winds buff.

To test further:
4: I had the other Shaman run 40+ yards away and drop his WF Totem again to clear their WF Totem buff on me (my original totem was still up) and Hero Rotation started showing to cast spells other than WF Totem.

Thank you for your help :)
